card/lib/card/view/options.rb

```diff
@@ -76,38 +94,44 @@ def define_setter option_key
         end
       end
 
+      #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      # VOO API
+
+      # The following methods allow developers to read and write options dynamically.
+
+
       # There are two primary options hashes:
-      # - @normalized_options are determined upon initialization and do not
-      #   change after that.
-      # - @live_options are created during the "process" phase, and they can be
-      #   altered via the "voo" API at any time
 
+      # - @normalized_options are determined upon initialization and do not change after that.
+      # @return [Hash] options
       attr_reader :normalized_options
 
+      # - @live_options are dynamic and can be altered by the "voo" API at any time. Such alterations are
+      #   NOT used in stubs
+      # @return [Hash]
       def live_options
         @live_options ||= process_live_options
       end
 
-      # The following methods comprise the primary voo API.  They allow
-      # developers to read and write options dynamically
 
-      def items
-        live_options[:items] ||= {}
-      end
+      # Developers can also set most options directly via accessors, eg voo.title = "King"
+      #
+      # Note that a few options do not follow this pattern.
```

Comment:
Why?

Reply:
Thanks.  Forgot to finish that piece.  Good catch.